### PR TITLE
Added cache system for schema loader.

### PR DESCRIPTION
**Pull request by chhill-redhat**
### Summary
This PR allows the schema loader to cache external schemas which were fetched by Ajax calls. The cache is the browser's `localStorage` storage. Caching is opt-in feature. By default, the caches are valid only for the current calendar day. However, the consumer can pass an explicit cache buster (e.g., the cache buster can be a hash of the schemas, to invalidate caches when any schema changes).

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Is backward-compatible?    | ✔️
| Tests pass?   | ✔️
| Fixed issues  | None
| Updated README/docs?   | ✔️
| Added CHANGELOG entry?   | ❌

### CODE REVIEW

File: tests/codeceptjs/codecept.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   },
   "helpers": {
     "WebDriver": {
-      "url": "http://node:9001/tests/pages/",
+      "url": "http://127.0.0.1:9001/tests/pages/",
```
Comment (schmunk42) [inline tests/codeceptjs/codecept.json:10]: Please do not change these URLs, they need to run in the CI pipelines with Docker.

Comment (chhill-redhat) [reply tests/codeceptjs/codecept.json]: Sorry, this was part of a work-in-progress commit. Fixed now.

**Comment by chhill-redhat** [comment]
@schmunk42 , this is ready for review. Tests pass locally (unit and e2e).

**Comment by germanbisurgi** [comment]
Hi @chhill-redhat,

Because the whole functionality of the json-editor depends on the schema-loader component, this PR will take a little more time to review as usual. We will give you feedback as soon as possible.

**Comment by mariohernandez** [comment]
Hi @germanbisurgi wondering if there are any updates on this PR. Thanks.

**Comment by slucero** [comment]
Hi @germanbisurgi and @schmunk42, has there been any more opportunity to review and test this work to get it pushed forward?

What are the best next steps to get this pushed forward?

**Comment by schmunk42** [comment]
Sorry guys for the delay, we were just not able to find enough time to review.

@chhill-redhat 
From my side it would be fine, since it is fully BC-compatible in the default setting, right?
So no cache is used when not explicitly enabled.

**Comment by chhill-redhat** [comment]
@schmunk42 , no worries! You're right, it's BC by default, because the cache is disabled unless explicitly enabled.

**Review by schmunk42 — APPROVED** [review]

**Comment by schmunk42** [comment]
@chhill-redhat Available in `2.9.0-beta.0` - thank you!

**Comment by chhill-redhat** [comment]
Great. Thank you, @schmunk42 ! Really appreciate it.

**Pull request merged**